MAINTAINERS

```diff
@@ -0,0 +1,332 @@
+	O: Other
+
+
+General repository administration
+---------------------------------
+M: Eduard Grasa <eduard.grasa (at) i2cat.net>
+M: Vincenzo Maffione <v.maffione (at) nextworks.it>
+M: Marc Sune <marc.sune (at) bisdn.de>
+M: Sander Vrijders <sander.vrijders (at) intec.ugent.be>
+T: https://github.com/IRATI/stack.git
+
+
+Issue tracking
+--------------
+M: Eduard Grasa <eduard.grasa (at) i2cat.net>
+M: Vincenzo Maffione <v.maffione (at) nextworks.it>
+M: Marc Sune <marc.sune (at) bisdn.de>
+M: Sander Vrijders <sander.vrijders (at) intec.ugent.be>
+O: irati@freelists.org
+O: https://github.com/IRATI/stack
+
+
+Documentation
+-------------
+M: Marc Sune <marc.sune (at) bisdn.de>
+F: MAINTAINERS
+F: check_maintainers.py
+F: doc/
+
+
+Build System
+------------
+
+Kernel space
+M: Vincenzo Maffione <v.maffione (at) nextworks.it>
+O: kernel/
+
+User space
+M: Marc Sune <marc.sune (at) bisdn.de>
+O: librina/
+O: rinad/
+O: rina-tools/
+
+
+Kernel space
+------------
+
+Core
+M: Vincenzo Maffione <v.maffione (at) nextworks.it>
+F: linux/net/rina/buffer*
+F: linux/net/rina/connection*
+F: linux/net/rina/common*
+F: linux/net/rina/core*
+F: linux/net/rina/debug*
+F: linux/net/rina/default*
+F: linux/net/rina/du*
+F: linux/net/rina/dt*
+F: linux/net/rina/ipcp-factories*
+F: linux/net/rina/ipcp-instances*
+F: linux/net/rina/ipcp-utils*
+F: linux/net/rina/logs.h
+F: linux/net/rina/personality*
+F: linux/net/rina/pci*
+F: linux/net/rina/pdu.*
+F: linux/net/rina/qos*
+F: linux/net/rina/rds/
+F: linux/net/rina/rina.h
+F: linux/net/rina/rina-personality*
+F: linux/net/rina/sdu*
+F: linux/net/rina/utils*
+
+SDK
+M: Vincenzo Maffione <v.maffione (at) nextworks.it>
+F: linux/net/rina/policies*
+F: linux/net/rina/ps-factory*
+F: linux/net/rina/rina-default-plugin*
+
+Shim eth-vlan and RINA ARP
+M: Sander Vrijders <sander.vrijders (at) intec.ugent.be>
+F: linux/net/rina/ipcps/shim-eth-vlan*
+F: linux/net/rina/rinarp/
+
+Shim tcp-udp
+M: Sander Vrijders <sander.vrijders (at) intec.ugent.be>
+F: linux/net/rina/ipcps/shim-tcp-udp*
+
+Shim hv (and VMPI)
+M: Vincenzo Maffione <v.maffione (at) nextworks.it>
+F: linux/net/rina/ipcps/shim-hv*
+F: linux/net/rina/vmpi/
+
+Shim dummy
+M: Miquel Tarzan <miquel.tarzan (at) i2cat.net>
+F: linux/net/rina/ipcps/shim-dummy*
+
+Kernel IPC process (standard)
+M: Miquel Tarzan <miquel.tarzan (at) i2cat.net>
+F: linux/net/rina/ipcps/normal-ipcp*
+
+KFA
+M: Miquel Tarzan <miquel.tarzan (at) i2cat.net>
+F: linux/net/rina/kfa*
+
+Kernel IPCM
+M: Leonardo Bergesio <leonardo.bergesio (at) i2cat.net>
+F: linux/net/rina/kipcm*
+F: linux/net/rina/pidm*
+
+RMT
+M: Miquel Tarzan <miquel.tarzan (at) i2cat.net>
+F: linux/net/rina/rmt*
+F: linux/net/rina/pim*
+
+PFF
+M: Sander Vrijders <sander.vrijders (at) intec.ugent.be>
+F: linux/net/rina/pft*
+
+EFCP
+M: Leonardo Bergesio <leonardo.bergesio (at) i2cat.net>
+F: linux/net/rina/efcp*
+F: linux/net/rina/cidm*
+
+SDU protection
+M: Ondrej Rysavy <rysavy (at) fit.vutbr.cz>
```

Comment:
The contact for this should probably be me (Ondrej Lichtner, ilichtner (at) fit.vutbr.cz) since Ondrej Rysavy doens't touch the kernel. And yeah, du-protection.{c,h} should be part of SDU Protection.